### PR TITLE
Avoid blocking by csp

### LINKED ISSUE #95: inline script fails with csp
Hi,

The view file `saml_post.html.erb` contains inline javascript: https://github.com/saml-idp/saml_idp/blob/5a1345eb457b9149f5c6e7343254ebe344643521/app/views/saml_idp/idp/saml_post.html.erb#L7 
This is blocked by content security policy, if set

Thanks

### PR DESCRIPTION
**Pull request by mkamensky**
Closes #95

### CODE REVIEW

**Comment by jphenow** [comment]
Hey! If you're still looking to get this in (and if not no problem I can add it to my list to dust off) - it appears this is in Rails 5.2+ so we'll need to check if it's a method and could be good to at least verify it does work as expected in 5.2+

**Comment by mkamensky** [comment]
> Hey! If you're still looking to get this in (and if not no problem I can add it to my list to dust off) - it appears this is in Rails 5.2+ so we'll need to check if it's a method and could be good to at least verify it does work as expected in 5.2+

Sorry, I don't understand what is said here, what is in Rails 5.2?

**Comment by jphenow** [comment]
I don't see `csp_meta_tag` before Rails 5.2 - I could be wrong

**Comment by Zogoo** [comment]
@jphenow @mkamensky 
I think this view could be overridden in Rails side. I think most of Rails users are overridding controller with their controller and using little bit differently. Because, Not everyone users these paremeters in controller `params[:email].blank? && params[:password].blank?`.
So, I would say in there we can render own view in there. Like: `render template: 'idp/saml_idp/my_saml_post', layout: false`

But, it's very good point maybe this should be in the document as suggestion for rails users.

File: app/views/saml_idp/idp/saml_post.html.erb [hunk 1]
```diff
@@ -2,13 +2,27 @@
 <html>
   <head>
     <meta charset="utf-8">
+    <%= csp_meta_tag %>
```
Comment (mjobin-mdsol) [inline app/views/saml_idp/idp/saml_post.html.erb:5]: for some reason this method is only callable from rails 5.2
it breaks tests in all other releases

**Comment by Zogoo** [comment]
@mkamensky , @mjobin-mdsol 
So, I think we can close this pull request and add in to README about this issue for Rails developers.

**Comment by mkamensky** [comment]
> @mkamensky , @mjobin-mdsol
> So, I think we can close this pull request and add in to README about this issue for Rails developers.

Sorry for the late reply. I don't think this is a big issue, but I'd rather have no views provided at all than something that doesn't work, and I have to figure out why. When I first ran into it, it took me  a while to figure out what's going on. I'm not sure I'd think about looking in the README for the solution. So my opinion is that it's better to provide nothing in this case, but as I said, not a big issue.

**Comment by Zogoo** [comment]
@mkamensky actually, I'm suggesting that (nothing in view) to other guys. Because, views and controllers aren't working well for most of the cases and it makes that more or less they need to update these resources by themselves.

Let me close this ticket and let me work on my suggestion with my next pull request.

**Pull request closed without merging**